hvac/api/system_backend/tools.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from hvac import utils
+from hvac.api.system_backend.system_backend_mixin import SystemBackendMixin
+
+
+class Tools(SystemBackendMixin):
+    def generate_random_bytes(
+        self, n_bytes=None, output_format=None, source=""
```

Comment:
Please default `source` to `None` and handle the templating needs in the method conditionally.